addons/escoria-core/game/core-scripts/esc/esc_room_manager.gd

```diff
@@ -0,0 +1,32 @@
+extends Object
+class_name ESCRoomManager
```

Comment:
I am not so fond of the name of this class: ESCRoomManager seems to imply this class manages rooms, but this is not true as it's just an enum entries holder. Would `ESCReservedGlobals` fit instead?

Reply:
Actually, my idea was that this manager would also change the room for example. Also, why's the GLOBAL_ANIMATION_RESOURCES here? That doesn't have to do anything with rooms if I'm not mistaken.

Reply:
Ah, I see. You saw it in ESCRoom and put it here. It's actually the global cache of the current animation resource of the player, so it doesn't really fit here. Hm... maybe ESCPlayer? I'm not 100% sure.

Reply:
> Actually, my idea was that this manager would also change the room for example

Can you give more precisions on this?

Reply:
> 

Yeah, you got it. It's also only ever used in ESCRoom, it seems. But I'm happy to put it wherever you deem best.

Reply:
> Ah, I see. You saw it in ESCRoom and put it here. It's actually the global cache of the current animation resource of the > player, so it doesn't really fit here. Hm... maybe ESCPlayer? I'm not 100% sure.

Yeah, you got it. It's also only ever used in ESCRoom, it seems. But I'm happy to put it wherever you deem best.

Reply:
Isn't it also used in the set_animations command? 🤔

Reply:
Actually, good catch. I didn't see it because it's actually referenced by a one-off constant in `globals_manager`, so I'll need to fix that anyway.

But like I said: I'll put the definition in whichever class you prefer!